src/modules/transaction/types.ts

```diff
@@ -105,3 +113,8 @@ export type Log = {
   name: string
   [key: string]: any
 }
+
+export type TransactionsConfig = {
+  squidUrl: string
```

Comment:
I know it's not that abstract right now to plug&plug another provider, but should we try to keep it generic and use something like `crossChainProviderURL`? I know we have `SQUID_URL` as the config env and Axelar as the provider name, but I'm just trying to not keep incorporating any of those names in here, wdyt?

Reply:
I've changed it to `crossChainProviderUrl` to make it more generic.